### PR TITLE
Add lib style down and implement base styleguid page

### PR DESCRIPTION
**Pull request by lossen**

### CODE REVIEW

File: src/scss/lib/texts.scss [hunk 1]
```diff
@@ -26,6 +26,12 @@
...
 
+.subtitle-large{
+  font-size: 18px;
+  font-weight: 400;
+  line-height: 21px;
+}
+
 .subtitle-link{
   font-family: $font-menu,sans-serif;
   color: #ffffff;
```
Comment (tiero) [inline src/scss/lib/texts.scss]: We should use variables for the main colors hex codes.

File: gulpfile.js [hunk 2]
```diff
@@ -11,12 +11,15 @@ const gulp      = require('gulp'),
...
+  imagemin      = require('gulp-imagemin'),
   size          = require('gulp-filesize'),
   gulpStylelint = require('gulp-stylelint'),
   minifyCSS     = require('gulp-minify-css'),
   autoprefixer  = require('gulp-autoprefixer'),
-  browserSync   = require('browser-sync').create();
+  browserSync   = require('browser-sync').create(),
+  shell = require('gulp-shell');
+
+gulp.task('generate-styleguid', shell.task('styledown src/styleguid/config.md > src/styleguid/styleguide.html'))
```
Comment (tiero) [inline gulpfile.js]: I would like to put it in a `public` folder at the root of the project so
- src
- scripts
- public

File: src/scss/lib/texts.scss (hunk 1 above)
Comment (lossen) [reply src/scss/lib/texts.scss]: Fixed

File: gulpfile.js (hunk 2 above)
Comment (lossen) [reply gulpfile.js]: fixed

**Comment by tiero** [comment]
rebase you master fork according to the last commit 411bf88

**Pull request merged**